processing/src/test/java/org/apache/druid/query/aggregation/first/FloatFirstAggregationTest.java

```diff
@@ -183,6 +201,40 @@ public void testSerde() throws Exception
     Assert.assertEquals(floatFirstAggregatorFactory, mapper.readValue(doubleSpecJson, AggregatorFactory.class));
   }
 
+  @Test
+  public void testFloatFirstAggregateCombiner()
+  {
+    AggregateCombiner floatFirstAggregateCombiner = combiningAggFactory.makeAggregateCombiner();
+
+    SerializablePair[] inputPairs = {
+        new SerializablePair<>(5L, 134.3f),
```

Comment:
can you also add tests with null values as input and/or expected result?

Reply:
Sorry for the late response. null related tests have been added in the latest commit